themes/powerline/powerline.base.sh

```diff
@@ -161,8 +162,9 @@ function __powerline_prompt_command {
   done
   [[ "${last_status}" -ne 0 ]] && __powerline_left_segment $(__powerline_last_status_prompt ${last_status})
   [[ -n "${LEFT_PROMPT}" ]] && LEFT_PROMPT+="$(set_color ${LAST_SEGMENT_COLOR} -)${separator_char}${_omb_prompt_normal}"
+  [[ "${POWERLINE_PROMPT_NEWLINE=}" == true ]] && END_PROMPT=" \n" || END_PROMPT=""
```

Comment:
I'm going to apply the following changes.

```suggestion
  [[ ${POWERLINE_PROMPT_NEWLINE-} == true ]] && END_PROMPT="\n" || END_PROMPT=" "
```

- I think we do not need a space before the newline. The newline was needed to make a space between the prompt and the command string, but now we have a newline after the prompt so the command is anyway in the beginning of the line.
- When we do not add the newline, we still need a space as before.
- In the conditional commands, the arguments are not needed to be quoted (except for the right-hand sides of `==`, `!=`, or `=~`)
- I think we do not need to make the variable `POWERLINE_PROMPT_NEWLINE` empty when it is not set. We can leave the unset state of the variable.